brands/amenity/fast_food.json

```diff
@@ -959,6 +959,18 @@
       "takeaway": "yes"
     }
   },
+  "amenity/fast_food|Jimmy the Greek": {
+    "countryCodes": ["ae", "ca"],
+    "tags": {
+      "amenity": "fast_food",
+      "brand": "Jimmy the Greek",
+      "brand:wikidata": "Q1689380",
```

Comment:
Should be Q17077817
(Q1689380 points to "Jimmy John's")

Reply:
fixed